lib/testing-helpcenter/help_center_tags.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Get array of tags from the https://helpcenter.onlyoffice.com/tags.aspx
```

Comment:
In future this class will hold no only `get array of tags` but some other methods as well, so description will be

```
Methods to work with tag list on https://helpcenter.onlyoffice.com/tags.aspx
```